flowclient/flowclient/client.py

```diff
@@ -1022,3 +1022,31 @@ def total_network_objects(
         "end_date": end_date,
         "aggregation_unit": aggregation_unit,
     }
+
+
+def aggregate_network_objects(
```

Comment:
I've been giving the params for this some thought - I actually think what would be most useful is to have it take: network_objects,  statistic, by. That can be achieved without touching the underlying Flowmachine class, by constructing a tno object, and then calling aggregate on it with the by and statistic arguments.

Reply:
You'll have to expand on that a bit, in terms of roughly how the call would look. I gather tno is TotalNetworkObject, and "by" is possibly the date range?! But also, I thought you were trying to do away with calls to aggregate() (#599)

Reply:
As you were, I understand now what you are driving at! (in view of the review comment further down)

Reply:
Arguments need to be `total_network_objects_query: dict`, and `by: str` (or a more explanatory name than `by` would also be fine!)